openvpn.js

```diff
@@ -29,12 +30,16 @@ const mkClient = (data, original = {}) => {
 
 const STATE = {idle: Symbol('idle'), status: Symbol('status')}
 class client extends EventEmitter {
-  constructor(host, port, pwd = null) {
+  constructor(host, port, pwd = null, netmask = null) {
     super()
     this.state = STATE.idle
     this.host = host
     this.port = port
     this.pwd = pwd
+    this.netmask = new Netmask ('0.0.0.0/0')
+    if (netmask != null) {
+            this.netmask = new Netmask(netmask);
```

Comment:
Some sort of validation would be great here. Maybe a regex?

Reply:
Don't want to reimplement Netmask checks. Catch exception instead.